src/components/ArticleDisplay.jsx

```diff
@@ -33,8 +70,20 @@ const ArticleDisplay = ({ article_id }) => {
       <ul className="article-display">
         <Article article={article} key={article.article_id} />
       </ul>
-      <button className="upvote-button">Upvote</button>
-      <button className="downvote-button">Downvote</button>
+      <button
+        onClick={() => upVote(article.article_id)}
+        className="upvote-button"
+      >
+        Upvote
+      </button>
+      <button
+        onClick={() => downVote(article.article_id)}
+        className="downvote-button"
+      >
+        Downvote
+      </button>
+      <p>{isUpVoteError ? "Could not upvote comment!" : ""}</p>
```

Comment:
These need to be bigger and bolder, a p tag does not have much in the way of visual hierarchy to tell me there has been a problem.